covalent_dispatcher/_cli/service.py

```diff
@@ -253,7 +253,7 @@ def _graceful_shutdown(pidfile: str) -> None:
     required=False,
     is_flag=False,
     type=str,
-    default="auto",
+    # default="auto",
```

Comment:
are the defaults "" for strings and 0 for ints?  If so you can remove these commented lines

Reply:
the default needed for `mem_per_worker` is "auto" and `threads_per_worker =1`, and `num_workers=dask.system.CPU_COUNT`